helpers/createArticleHelper.js

```diff
@@ -11,8 +11,9 @@ import { Article, User } from '../models';
  */
 
 const createArticleHelper = (res, articleObject, imageUrl = null) => {
+  articleObject.price = (articleObject.price) ? articleObject.price.toFixed(2) : undefined;
```

Comment:
why make this `undefined` rather than 0?

Reply:
My approach before was to leave it as `undefined` if it is not defined, so far `isPaidFor` is not set to `true` but I realize now that making it `0` makes better sense. I will fix it, thanks